### DISCUSSION TITLE
Increment variable in jslt

### DISCUSSION THREAD

**Discussion by abhishekrongali** [schibsted/jslt#314] [General]
Hi,

Is there a way to increment a variable defined in jslt?

Like I declared a variable in jslt like -> let count = 0
I want to increment it in next lines of codes when specific condition satisfies it, can you please help me with sample examples?

Thanks in advance!

**Comment by larsga** [comment]
In short, no. JSLT doesn't let you modify "variables".

What you can do is build an array that contains only the objects that satisfy the condition, then count the length of it. Or you can use a recursive function for the computation.

You have to think in a functional style with JSLT. It's not an imperative language like Java.
